cadquery/occ_impl/shapes.py

```diff
@@ -730,6 +730,47 @@ def makeCircle(cls, radius, pnt=Vector(0, 0, 0), dir=Vector(0, 0, 1), angle1=360
                                              True).Value()
             return cls(BRepBuilderAPI_MakeEdge(circle_geom).Edge())
 
+    @classmethod
+    def makeEllipse(cls, x_radius, y_radius, pnt=Vector(0, 0, 0), dir=Vector(0, 0, 1),
+                    angle1=360.0, angle2=360.0):
+        """
+        Interpolate a spline through the provided points.
+        :param cls:
+        :param x_radius: x radius of the ellipse (along the x-axis of plane the ellipse should lie in)
+        :param y_radius: y radius of the ellipse (along the y-axis of plane the ellipse should lie in)
+        :param pnt: vector representing the center of the ellipse
+        :param dir: vector representing the direction of the plane the ellipse should lie in
+        :param angle1: start angle of arc
+        :param angle2: end angle of arc (angle2 == angle1 return closed ellipse = default)
+        :return: an Edge
+        """
+
+        pnt = Vector(pnt).toPnt()
```

Comment:
Could you make it so that pnt and dir could be tuples here as well? There's a discussion about how we're not handling Vector vs tuple consistently here: https://github.com/CadQuery/cadquery/issues/256

Reply:
Hold on, this isn't end-user facing at the top level of the abstraction, so it probably doesn't matter whether they Vector or tuple.

Reply:
The signature of `makeEllipse` is a copy of `makeCircle` and afaik both `cq.Vector(tuple)` and `cq.Vector(vector)` return a `Vector`. So `Vector(pnt)` and `Vector(dir)` will return `Vector` types even when a tuple is passed as a parameter